samples/cpp/arrow/readme.html

```diff
@@ -930,34 +930,73 @@ <h1 class="title">Intel® Data Analytics Acceleration Library Apache Arrow* Samp
 the Apache Arrow* library in a C++ application.</p>
   <p>Unzip the archive with Intel® DAAL samples to your working directory (<span class="filepath"><var>&lt;sample_dir&gt;</var></span>).
 <h2><a name="System_Requirements">System Requirements</a></h2>
-<p>You can use Intel® DAAL Apache Arrow samples on Linux* operating system.
+<p>You can use Intel® DAAL Apache Arrow* samples on Linux*, Windows* and MacOS* operating systems.
 For a list of Intel® DAAL hardware and software requirements, refer to release notes
 for the version of Intel® DAAL you are using. </p>
     <h3>Apache Arrow* implementations against which Intel® DAAL has been validated:</h3>
     <ul>
-        <li>Apache Arrow* 0.13.0</li>
+        <li>Apache Arrow* 0.15.0</li>
```

Comment:
We have 0.15.1 already.

Reply:
Sample was successfully built and tested with 0.15.1 version. Readme files were changed.